cpp/src/binaryop/jit/code/kernel.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Copyright 2018-2019 BlazingDB, Inc.
+ *     Copyright 2018 Christian Noboa Mardini <christian@blazingdb.com>
+ *     Copyright 2018 Rommel Quintanilla <rommel@blazingdb.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace cudf {
+namespace experimental {
+namespace binops {
+namespace jit {
+namespace code {
+
+const char* kernel =
+R"***(
+    #include <cudf/types.hpp>
+    #include <simt/limits>
+    #include <cudf/wrappers/timestamps.hpp>
+    #include "operation.h"
+
+namespace cudf {
+namespace experimental {
```

Comment:
I'm not sure. The type names need to be fully qualified because Jitify's invocation of this function isn't in this namespace, and can't find the types with which to invoke the kernel.